malaria24/ona/tests/test_models.py

```diff
@@ -0,0 +1,111 @@
+import pkg_resources
+
+from django.test import TestCase, override_settings
+from django.utils import timezone
+
+from testfixtures import LogCapture
+
+import responses
+
+from malaria24.ona.models import ReportedCase, EHP, SMS
+
+
+@override_settings(CELERY_ALWAYS_EAGER=True)
+class ReportedCaseTest(TestCase):
+
+    def setUp(self):
+        responses.add(
+            responses.PUT,
+            ('https://go.vumi.org/api/v1/go/http_api_nostream/'
+             'VUMI_GO_CONVERSATION_KEY/messages.json'),
+            status=200, content_type='application/json',
+            body=pkg_resources.resource_string(
+                'malaria24', 'ona/fixtures/responses/send_sms.json'))
```

Comment:
`responses` is a library that allows you to mock HTTP requests made with the excellent `requests` library. This way you can specify a fixed responses that needs to be returned when an HTTP call is made to some place. In this case I'm faking the response from an API call to Vumi's HTTP API.